### ISSUE TITLE
connecting to another site

### ISSUE THREAD

**Issue by russgove** [pnp/pnpjs#1676]
### Category
- [ ] Enhancement
- [ ] Bug
- [x] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [ 2.3.0        ]

Please specify what version(s) of SharePoint you are targeting: [SPO]

Hi,
haven't worked wit pnpjs in a bit and am running into an issue.

I have this code:
export async function createCaptureList(client: AadHttpClient, siteUrl: string, listName: string, managementApiUrl: string): Promise<any> {
    debugger;
    var url: string = decodeURIComponent(siteUrl);
    var rootweb = Web(url);
    try {
       rootweb.lists.getByTitle(listName);
        throw new Error("List already exists");
    }
    catch (e) {
    }
    const newList = await rootweb.lists.add(listName, "Audit Data", 100, true);
}
basically create a list in some site if it doesnt exist. 

The issue is that after i create the rootweb variable from the pnpjs call it dose not have a lists attribute.
Viwing rootweb in debugger shows this:
![image](https://user-images.githubusercontent.com/2412956/113491421-a6da4c80-949e-11eb-9f8c-3d2ce6db7c55.png)

What did i do wrong?

**Comment by juliemturner** [comment]
I suspect your missing some imports, but I could be wrong since I'm only looking at a partial code snippet... check [the docs](https://pnp.github.io/pnpjs/sp/lists/#ilists) and validate that against the imports you have in your file...

**Comment by russgove** [comment]
yup. adding presets/all fixed it.
thanks!

**Comment by juliemturner** [comment]
It does, but that bloats your build package just so you're aware. Always our recommendation to use the selective imports.
